### ISSUE TITLE
Add a program option to modify the commit rate

### ISSUE THREAD

**Issue by studiofuga** [frabas/DISPLACE_GUI#113]
Commit rate in simulation is hardwired to 100, it would be nice to have the possibility to fine tune it with a proper command line option:

`--commit-rate 250` Will commit the changes every 250 steps.
Needs `boost::program_option` so it depends on #87
